datalad/support/gitrepo.py

```diff
@@ -697,7 +710,7 @@ def __eq__(self, obj):
     @classmethod
     def is_valid_repo(cls, path):
         """Returns if a given path points to a git repository"""
-        return exists(opj(path, '.git', 'objects'))
+        return exists(opj(path, '.git'))
 
```

Comment:
The most important reason is, that a repository with .git file was not accepted as valid by this method, since there is `.git/objects` in that case. We could read the file and construct the path to `objects` of course - would be a little bit more expensive.

Good enough - I think so. For now it doesn't break anything and: We also used the constructor of GitPython's `Repo` to determine whether a repo is valid. And this one does just this: Looking for a `.git`